docs/user/getting_started.md

```diff
@@ -0,0 +1,216 @@
+```
+* Enter the mizar directory and run the bootstrap.sh script.
+    * This script will the neccessary components to compile Mizar, and run it's unit tests. These include
+        * Python
+        * Docker
+        * Build Essentials
+        * Cmocka
+        * Clang-7
+        * Llvm-7
+    * There is also an optional kernel update included in the script if you wish to update to a compatible version
+
+```
+$ ./bootstrap.sh
+```
+
+
+## Setup
+
+### Compiling Mizar
+
+After completing the prerequisites above, you can now compile mizar. Enter the mizar directory and run the command below.
+
+```
+$ make
+```
+
+* You can now optionally run the unit tests and coverage reports by running the commands below.
+
+```
+$ run_unittests
+$ make lcov
+```
+
+### New Kind Cluster
+
+If you are testing out Mizar with Kind, a script has been included to setup a local Kubernetes cluster, and install all of the components needed to run Mizar.
+Simply run the script below in the Mizar directory.
+
+```
+$ ./kind_setup.sh
+```
+
+This script does the following:
+
+* Create a multi-node local Kubernetes cluster with Kind.
+* Build all of the necessary Docker images.
+* Apply all of the Mizar CRDs.
+* Deploy the Mizar Operators
+* Install the Mizar CNI
+
+### Existing Kubernetes Cluster
+
+If you are installing Mizar on an existing Kubernetes cluster, please complete the following steps.
+
+To be expanded...
+
+## Running Mizar Management Plane
+
+When a cluster is brought up with Mizar, a default VPC and default Network are created. We will call these **vpc0** and **net0**. The default VPC will have an initial default number of Dividers provisioned. Similarly the default Network will also have an initial default number of Bouncers provisioned. These resources will automatically scale in and out with depending on the load of the cluster. A user may also manually provision more Bouncers and Dividers at will to fit their needs.
+
+### Inspecting the resources.
+
+You can inspect the following resources by running the commands below with ```kubectl```.
+
+ * Get all VPCs in the cluster
+
+```kubectl get vpcs```
+
+ * Get all Networks in the cluster
+
+```kubectl get nets```
+
+ * Get all Dividers in the cluster
+
+```kubectl get dividers```
+
+ * Get all Bouncers in the cluster
+
+```kubectl get bouncers```
+
+ * Get all endpoints in the cluster
+
+```kubectl get endpoints```
+
+
+### Creating an endpoint
+
+Now that the steps above have been completed, simply create a deployment of any kind to test out the network functionality of Mizar.
+A sample deployment with nginx has been included below.
+
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 10
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:1.7.9
+          ports:
+            - containerPort: 80
+```
```

Comment:
You can run ping test from a Nginx machine: 

kubectl exec -it nginx-deployment-your-nigix-machine /bin/bash 

Now, you are running in interactive mode inside the nginx docker container. From the docker container, ping a different host: 

ping 10.0.0.5

Reply:
I can add this to be beginner friendly.